Update smartcar rev2.ino

AlbertSimpson00 opened the pull request:
basic obstacle avoidance, stops the car if reaches within 20cm of an object

AlbertSimpson00 committed:
Update smartcar.ino

basic obstacle avoidance, stops the car if reaches within 20cm of an object

platisd commented on arduino/smartcar/smartcar.ino:
```diff
@@ -1,9 +1,48 @@
+}
+
+void stop() {
+    if(front.getDistance() > 20) {
```
> basic obstacle avoidance, stops the car if reaches within 20cm of an object

That's not what this line does.

AlbertSimpson00 committed:
Update smartcar.ino

made changes to make the code actually do as stated:
-car stops within 1m
-then automatically goes in reverse to provide some space for next command input
